Backend/ActiveCruzer/Controllers/UserController.cs

```diff
@@ -125,6 +130,53 @@ public ActionResult<JwtDto> Login([FromBody] CredentialsDTO credentials)
 
             return Unauthorized();
         }
+        /// <summary>
+        /// Delete user account
+        /// </summary>
+        /// <param name="userId"></param>
+        /// <returns></returns>
+        [Authorize]
+        [HttpDelete]
+        [Route("Delete")]
```

Comment:
please specify the possible response codes

Reply:
done